storagemarket/dealstatus.go

```diff
@@ -141,3 +141,37 @@ var DealStates = map[StorageDealStatus]string{
 	StorageDealClientTransferRestart:        "StorageDealClientTransferRestart",
 	StorageDealProviderTransferAwaitRestart: "StorageDealProviderTransferAwaitRestart",
 }
+
+// DealStatesDescriptions maps StorageDealStatus codes to string description for better UX
+var DealStatesDescriptions = map[StorageDealStatus]string{
```

Comment:
I'd merge this with the above map, making it a `map[StorageDealStatus]struct{Name: string, Desc: string}`

Reply:
The DealStates map is already used in a lot of places across the code so if you unify the maps, may be a good idea to add an init() function that also recreates the DealStates map

Reply:
> The DealStates map is already used in a lot of places across the code

This was the main reason I created a new map.

I suggest we do that refactor as a next step, as I am sure we will want to adjust messages and descriptions very soon.